Cubical/Categories/NaturalTransformation/Base.agda

```diff
@@ -0,0 +1,271 @@
+      βy'≡βy = symP (toPathP {A = λ i → D [ Gy≡G'y (~ i) , H ⟅ y ⟆ ]} refl)
+
+      βx≡βx' : PathP (λ i → D [ Gx≡G'x (~ i) , H ⟅ x ⟆ ]) (β ⟦ x ⟧) βx'
+      βx≡βx' = toPathP refl
+
+      -- left wall of square
+      left : PathP (λ i → D [ Gx≡G'x i , H ⟅ y ⟆ ]) (G ⟪ f ⟫ ⋆⟨ D ⟩ βy') (G' ⟪ f ⟫ ⋆⟨ D ⟩ β ⟦ y ⟧)
+      left i = Gf≡G'f i ⋆⟨ D ⟩ βy'≡βy i
+
+      -- right wall of square
+      right : PathP (λ i → D [ Gx≡G'x (~ i) , H ⟅ y ⟆ ]) (β ⟦ x ⟧ ⋆⟨ D ⟩ H ⟪ f ⟫) (βx' ⋆⟨ D ⟩ H ⟪ f ⟫)
+      right i = βx≡βx' i ⋆⟨ D ⟩ refl {x = H ⟪ f ⟫} i
+
+      -- putting it all together
+      βSq : G ⟪ f ⟫ ⋆⟨ D ⟩ βy' ≡ βx' ⋆⟨ D ⟩ H ⟪ f ⟫
+      βSq i = comp (λ k → D [ Gx≡G'x (~ k) , H ⟅ y ⟆ ])
+                   (λ j → λ { (i = i0) → left (~ j)
+                            ; (i = i1) → right j })
+                   (β .N-hom f i)
+
+  -- hmm are these judgementally equal when refl?
+  -- nope
+  -- does that matter? is there way to make it judgementally equal?
+  -- hmm : {F G H : Functor C D} {α : NatTrans F G} {β : NatTrans G H}
+  --     → seqTrans α β ≡ seqTransP {p = refl} α β
+  -- hmm = {!refl!}
+
+
+  module _  ⦃ isCatD : isCategory D ⦄ {F G : Functor C D} {α β : NatTrans F G} where
+    open Precategory
+    open Functor
+    open NatTrans
+
+    makeNatTransPath : α .N-ob ≡ β .N-ob → α ≡ β
+    makeNatTransPath p i .N-ob = p i
+    makeNatTransPath p i .N-hom f = rem i
+      where
+        rem : PathP (λ i → (F .F-hom f) ⋆ᴰ (p i _) ≡ (p i _) ⋆ᴰ (G .F-hom f)) (α .N-hom f) (β .N-hom f)
+        rem = toPathP (isCatD .isSetHom _ _ _ _)
+
+  module _  ⦃ isCatD : isCategory D ⦄ {F F' G G' : Functor C D}
+            {α : NatTrans F G}
+            {β : NatTrans F' G'} where
+    open Precategory
+    open Functor
+    open NatTrans
+    makeNatTransPathP : ∀ (p : F ≡ F') (q : G ≡ G')
+                      → PathP (λ i → (x : C .ob) → D [ (p i) .F-ob x , (q i) .F-ob x ]) (α .N-ob) (β .N-ob)
+                      → PathP (λ i → NatTrans (p i) (q i)) α β
+    makeNatTransPathP p q P i .N-ob = P i
+    makeNatTransPathP p q P i .N-hom f = rem i
+      where
+        rem : PathP (λ i → ((p i) .F-hom f) ⋆ᴰ (P i _) ≡ (P i _) ⋆ᴰ ((q i) .F-hom f)) (α .N-hom f) (β .N-hom f)
+        rem = toPathP (isCatD .isSetHom _ _ _ _)
+
+
+
+private
+  variable
+    ℓA ℓA' ℓB ℓB' : Level
+
+module _ {B : Precategory ℓB ℓB'} {C : Precategory ℓC ℓC'} {D : Precategory ℓD ℓD'} where
+  open NatTrans
+  -- whiskering
+  -- αF
+  _∘ˡ_ : ∀ {G H : Functor C D} (α : NatTrans G H) → (F : Functor B C)
+        → NatTrans (G ∘F F) (H ∘F F)
+  (_∘ˡ_ {G} {H} α F) .N-ob x = α ⟦ F ⟅ x ⟆ ⟧
+  (_∘ˡ_ {G} {H} α F) .N-hom f = (α .N-hom) _
+
+  -- Kβ
+  _∘ʳ_ : ∀ (K : Functor C D) → {G H : Functor B C} (β : NatTrans G H)
+       → NatTrans (K ∘F G) (K ∘F H)
+  (_∘ʳ_ K {G} {H} β) .N-ob x = K ⟪ β ⟦ x ⟧ ⟫
+  (_∘ʳ_ K {G} {H} β) .N-hom f = preserveCommF {C = C} {D = D} {K} (β .N-hom f)
+
+
+
+-- module _ (C : Precategory ℓC ℓC') (D : Precategory ℓD ℓD') ⦃ isCatD : isCategory D ⦄ where
```

Comment:
Why keep all of this commented code?

Reply:
Ah that's left over from refactoring. My bad